_posts/2022-08-17-#25665.md

```diff
@@ -7,18 +7,40 @@ authors: [ryanofsky]
 components: ["refactoring"]
 host: stickies-v
 status: upcoming
-commit:
+commit: 590bc615a
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- This PR is a followup to [#25218](https://github.com/bitcoin/bitcoin/pull/25218) introducing helper class `BResult`, which we discussed in a [previous review club]((https://bitcoincore.reviews/25218)).
+
+- In [#25721](https://github.com/bitcoin/bitcoin/pull/25721) `BResult` was renamed to `util::Result`, and the interface modified to resemble that of `std::optional`.
+
+- The description of this latest PR [#25665](https://github.com/bitcoin/bitcoin/pull/25665) comprehensively lists the introduced changes, as well as useful information on history and alternatives. In summary, the proposed functional changes compared to the initial `BResult` design are to:
```

Comment:
Sorry I havent actually reviewed the PR, but this sounds similar to the rust result module https://doc.rust-lang.org/std/result/, can link that if relevant?

Reply:
Good idea, I've seen people compare it with the rust implementation on earlier PRs. Will add link.